### PR TITLE
About screen add system images for accessory view

### PR DESCRIPTION
**Pull request by bozidarsevo**
## Description

- "Added accessoryViewSystemImage to AboutItem" in https://github.com/Automattic/AutomatticAbout-Swift/pull/9
- Using that change to add icons next to rate and share rows
- renamed "Twitter" to "X"

## Testing instructions

- Open Menu
- Open Settings
- Scroll to "About the app" and tap "WooCommerce"
- Check that "Rate us" and "Share with Friends" have icons next to it
- Check that "Instagram" has "@woocommerce" for the subtitle
-

## Screenshots

| Before      | After |
| ----------- | ----------- |
| Simulator Screenshot - iPhone 15 Pro - 2024-03-12 at 16 44 32      | Simulator Screenshot - iPhone 15 Pro - 2024-03-15 at 17 16 26      |

---
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by soundsokay** [comment]
Consider renaming the label @bozidarsevo to reflect Twitter's rebranding:

"Twitter" -> "X"

**Comment by bozidarsevo** [comment]
> Consider renaming the label @bozidarsevo to reflect Twitter's rebranding:
> 
> 
> 
> "Twitter" -> "X"

Nice catch! Tnx!

File: WooCommerce/Resources/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -10047,8 +10047,8 @@ which should be translated separately and considered part of this sentence. */
 /* Info text about the scanned text list on the add product from image form. */
 "Tweak your text: Unselect scans you don't need or tap to edit" = "Tweak your text: Unselect scans you don't need or tap to edit";
 
-/* Title of a button linking to the app's Twitter profile */
-"Twitter" = "Twitter";
+/* Title of a button linking to the app's X profile */
+"X" = "X";
```
Comment (iamgabrielma) [inline WooCommerce/Resources/en.lproj/Localizable.strings]: AFAIK this will get translated as part as the release process, we should revert this change.

**Review by iamgabrielma — APPROVED** [review]
LGTM! 🚢 

Initially I though the "Rate" button was broken, as it returned _"Safari cannot open the page because the address is invalid",_ but this only happen in Xcode's simulator, it works as expected in physical device 👍 

Please be sure to revert the change to `Localizable.strings` before merging
